src/libguac/guacamole/user.h

```diff
@@ -89,6 +89,12 @@ struct guac_user_info {
      */
     int optimal_resolution;
     
+    /**
+     * The human-readable name of the Guacamole user. If the client does not
+     * provide a name then this will be NULL.
+     */
+    const char* username;
```

Comment:
New structure members should be added to the end of the `struct` to preserve binary compatibility of the offsets of existing structure members. If at the end, existing code would need only be rebuilt if it allocates `guac_user_info` itself.

Reply:
Got it - have moved it down.